scripts/zones/Phomiuna_Aqueducts/npcs/_0r9.lua

```diff
@@ -21,9 +19,6 @@ entity.onEventUpdate = function(player, csid, option, npc)
 end
 
 entity.onEventFinish = function(player, csid, option, npc)
-    if csid == 37 then
-        player:setCharVar('X_MARKS_THE_SPOT', 5)
```

Comment:
Interesting that these X_MARKS_THE_SPOT vars are completely vestigial, and not set/used anywhere else 🤷

Reply:
It was added to the code way back in 2020. My best guess is that someone was working on it but eventually scrapped it.